_posts/2020-09-01-note-scheduled-upgrades.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: A note on scheduled protocol upgrades
+summary: Elaboration of the reduction in frequency of scheduled protocol upgrades.
+tags: [community]
+author: dEBRUYNE & sgp with input and corrections from community members
```

Comment:
Would be better to have a shorter text here, since this field is not meant to display such long string. I and @dEBRUYNE-1  couldn't come up with something shorter with the same meaning. So leaving it as it is for now, unless somebody has a better solution.

Reply:
"dEBRUYNE, Justin Ehrenhofer, and community members"